WORKSPACE

```diff
@@ -30,13 +34,39 @@ load("@aspect_bazel_lib//lib:repositories.bzl", "register_coreutils_toolchains")
 register_coreutils_toolchains()
 
 ############################################
-## CC toolchain using zig
-load("@hermetic_cc_toolchain//toolchain:defs.bzl", zig_toolchains = "toolchains")
```

Comment:
nit: I think we could remove a fetch for this? will do in a follow-up, anyhow we're likely to change the cc toolchain again for musl